CONTRIBUTING.md

```diff
@@ -46,6 +44,4 @@ Please feel free to use the bot on your own issue or PR to add yourself as a con
 
 The docs are published at http://focus-trap.github.io/focus-trap/ and served from the `/docs` directory in the repo.
 
-Simply run `yarn demo-bundle` locally, commit the updated `/docs/demo-bundle.js` file, and open a PR to updated it.
-
-Ideally, this is done within any PR that modifies anything under `/docs`, or the library code itself.
```

Comment:
demo-bundle is build with build command so both at release and prepare. We should probably add it to gitignore. I kept it there to show the difference between browserify and rollup output

Reply:
Actually, we need to keep these instructions, and keep `/docs/demo-bundle.js`, because it's needed for the demo to run, and the demo is just static files in `/docs` which are served by GitHub. There's no "netlify auto-build, generate files, and host them" process happening behind the scenes...

Reply:
ok